Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require "rubocops/shared/string_content"
+require "rubocops/shared/helper_functions"
+
+# TODO: move the micro shared modules above into this one?
+
+module RuboCop
+  module Cop
+    module DescHelper
+      include StringContent
+      include HelperFunctions
+
+      VALID_LOWERCASE_WORDS = %w[
+        macOS
+      ].freeze
+
+      def audit_desc(type, name, desc_call)
+        # Check if a desc is present
+        if desc_call.nil?
+          problem "#{type.to_s.capitalize} should have a desc (Description)."
+          return
+        end
+
+        desc = desc_call.first_argument
+
+        # Check the desc length. Should be >0 and <80 characters.
+        pure_desc_length = string_content(desc).length
+        if pure_desc_length.zero?
+          problem "The desc (description) should not be an empty string."
+          return
+        end
+
+        # Check for leading whitespace.
+        problem "Description shouldn't have a leading space" if regex_match_group(desc, /^\s+/)
+
+        # Check for trailing whitespace.
+        problem "Description shouldn't have a trailing space" if regex_match_group(desc, /\s+$/)
+
+        # Check if command-line is wrongly used in the desc
+        if match = regex_match_group(desc, /(command ?line)/i)
+          c = match.to_s[0]
+          problem "Description should use \"#{c}ommand-line\" instead of \"#{match}\""
```

Comment:
Similarly to the comments, some of the error messages end with a period, others don't. We should add them consistently, right?

Reply:
Yeah, these should end with a period.

Reply:
Thanks, I've added the period (also took the opportunity to rephrase some of the messages for clarity) and adjusted the tests accordingly.